share/transformers/com_mendeley_data.py

```diff
@@ -0,0 +1,435 @@
+from share.transform.chain import *  # noqa
+from share.transform.chain.utils import format_address
+
+
+def format_mendeley_address(ctx):
+    return format_address(
+        address1=ctx['name'],
+        city=ctx['city'],
+        state_or_province=ctx['state'],
+        country=ctx['country']
+    )
+
+RELATION_MAP = {
+    'related_to': '',
+    'derived_from': 'IsDerivedFrom',
+    'source_of': 'IsDerivedFrom',
+    'compiles': 'Compiles',
+    'compiled_by': 'Compiles',
+    'cites': 'Cites',
+    'cited_by': 'Cites',
+}
+
+INVERSE_RELATIONS = (
```

Comment:
Nit-pick: If you're just using these to check membership a `set` is much more efficient than a `tuple`